app-admin/calamares/calamares-3.1.1.ebuild

```diff
@@ -7,9 +7,9 @@ PYTHON_COMPAT=( python{3_4,3_5} )
 inherit kde5 python-r1
 
 DESCRIPTION="Distribution-independent installer framework"
-HOMEPAGE="http://calamares.io"
+HOMEPAGE="https://calamares.io"
```

Comment:
This was a tough call, because it only expired less than two weeks ago, so I figured it may be likely that they will fix the expired certificate issue when they know of the problem. In other cases where certificates were clearly expired by long periods, had the wrong CN/SAN fields, etc I didn't switch those to https://

Any preference for how to handle this particular case?